tox.ini

```diff
@@ -2,42 +2,29 @@
 envlist =
 # Jython support pending 2.7 support, due 2012-07-15 or so.  See:
 # http://fwierzbicki.blogspot.com/2012/03/adconion-to-fund-jython-27.html
-    py27,py27-minimal,py27-pure,pypy,py33,py34,py34-pure,py35,py35-pure,pypy3,coverage,docs
+    py27,py27-minimal,py27-pure,pypy,py33,py34,py34-pure,py35,py35-pure,py36,pypy3,coverage,docs
```

Comment:
I debated about that. They're not useful by default, but someone could configure their `pip` in such a way as to make them useful. So maybe leave the configurations but take them out of the envlist? (Oh, that's exactly what you're suggesting, isn't it?)